tests/shap/test_shap.py

```diff
@@ -0,0 +1,83 @@
+import sklearn
+import sklearn.linear_model
+from sklearn.model_selection import train_test_split
+
+import pytest
+import shap
+
+import meteors as mt
+
+
+def simple_explanation_test():
```

Comment:
Test all `explainer_types` even None. This test is repeated in `test_shap_explain`. Additionally validate that explanation outputs are correct. You need to add some validation checks also.

Reply:
fixed in 447480814aa02bacc6bfad3c3b6e5cf912ad5fb2